CMakeLists.txt

```diff
@@ -36,11 +36,21 @@ FetchContent_Declare(
     SOURCE_SUBDIR llvm
 )
 
+FetchContent_Declare(
+    diff
+    GIT_REPOSITORY https://github.com/fosterbrereton/diff.git
```

Comment:
The code in this repository is a fork of a fork + additional modernizations. The header was released under the Apache 2.0 license. Although it seemed OK to me, IANAL, so I thought it best to isolate the code into its own repository and keep it separate from hyde.